helper/triples.php

```diff
@@ -39,6 +40,40 @@ function _expandTokens($str) {
     }
 
     /**
+     * Initializes SQL filter of blocked graphs (no page permission by ACL rules).
+     */
+     function init_blocked_graphs() {
+      if ($this->getConf('use_acl') and self::$blocked_graphs_sql == '') {
+        $sql = "SELECT DISTINCT graph FROM " . self::$readable;
+        
+        // prepare query
+        $query = $this->_db->prepare($sql);
+        if($query == false) return;
+
+        // execute query
+        $res = $query->execute($values);
+        if($res === false) {
+            $error = $query->errorInfo();
+            msg(sprintf($this->getLang('error_graphs_fetch'),hsc($error[2])),-1);
+        }
+
+        // fetch results and return them
+        $result = $query->fetchAll(PDO::FETCH_ASSOC);
+        $query->closeCursor();
+        
+        // a list of graphs reachable by the current user is generated 
+        $blocked_graphs = array();
+        foreach ($result as $row) {
+          if (auth_quickaclcheck($row['graph']) < AUTH_READ)
```

Comment:
To improve maintainability of the code, please use `{` and `}` when you stretch an if statement over multiple lines, even when the body is only a single statement.